src/ArcgisAccount/useAccountManager.js

```diff
@@ -0,0 +1,189 @@
+import { useState, useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  completeAuth,
+  loginOAuth2,
+  logoutOAuth2,
+  getUserThumbnail,
+  getOrgThumbnail
+} from './accountManagerUtils';
+
+import {
+  getAccountManagerStorage,
+  addAccountStorage,
+  removeAccountStorage,
+  logoutAccountStorage,
+  beginStatusStorage,
+  switchActiveStorage,
+  completeStatusStorage,
+  refreshAccountStorage
+} from './accountManagerStorageUtils';
+
+const useAccountManager = props => {
+  const { accountManagerName, options } = props || {};
+  const [name] = useState(
```

Comment:
@brandonlammey Is state necessary if we're never setting it anywhere else?

Reply:
Updated